CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+- Addition of markdown support for headline and subhead of infocard component. 
```

Comment:
Please adhere to the keep a changelog format: https://keepachangelog.com/en/1.0.0/

```suggestion
### Added
- Markdown support for headline and subhead of infocard component. 
```